### DISCUSSION TITLE
Is there a formal registration process for the 2023-2024 Challenge?

### DISCUSSION THREAD

**Discussion by juseonghan** [surgical-robotics-ai/surgical_robotics_challenge#88] [Q&A]
I clicked on "Registration Form" on the 2023 challenge website at https://surgical-robotics-ai.github.io/surgical-robotics-challenge-2023/challenge-2023.html. However, it doesn't do anything. Is there a formal registration process, or has registration closed already for the challenge this year? Thank you.

**Comment by pkazanzides** [comment]
The registration link is now valid.
